acceptance-tests/basic_pcap_test.go

```diff
@@ -0,0 +1,68 @@
+package acceptance_tests
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gexec"
+	"net/http"
+	"os"
+	"os/exec"
+	"time"
+)
+
+var _ = Describe("Pcap Deployment", func() {
+	It("Deploys successfully", func() {
+
+		info, _ := deployPcap(
+			baseManifestVars{
+				deploymentName: deploymentNameForTestNode(),
+			},
+			[]string{},
+			map[string]interface{}{},
+			true,
+		)
+
```

Comment:
We could consider moving this into the `Describe` block above, or a `Context`.

This is of course unless the different tests within a single test file would be executed on different nodes with different `deploymentNameForTestNode()`?

Reply:
>This is of course unless the different tests within a single test file would be executed on different nodes with different deploymentNameForTestNode()?

Yes, they are. All `It`s are randomized and run in parallel. (we only have one here, but you get the point :)
Each test means a different deployment name and we have several run in parallel so each needs a different name to avoid clashes.

Reply:
But with `BeforeEach` you'd still have it in the context of the same "thread", i.e.
1. `BeforeEach`
2. `It`
3. (`AfterEach`)

Reply:
yes, makes sense when there is more than one test.